.github/workflows/build.yaml

```
@@ -14,21 +17,18 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-    env:
-      CIBW_SKIP: 'cp27-* cp35-* pp*'
-      CIBW_ARCHS_LINUX: auto aarch64
     steps:
+      - uses: actions/checkout@v2
       - name: Set up QEMU
         if: runner.os == 'Linux'
         uses: docker/setup-qemu-action@v1
         with:
           platforms: arm64
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: '3.9'
-      - run: pip install cibuildwheel
-      - run: cibuildwheel
+      - uses: joerick/cibuildwheel@v1.9.0
+        env:
+          CIBW_BUILD: cp36-*
```

```suggestion
          CIBW_BUILD: cp36-* cp39-macosx_universal2
```

Hopefully I got the name right, we forgot to add it to our table. 😳

---

I'm curious to see if this a) produces a reasonably named wheel, and b) is correctly loaded by Pip; supporting 10.9 universal was something they forgot to consider, cp36 universal sounds like something that might be easy to miss, too. :)

---

You can also add `CIBW_TEST_SKIP: *universal2:arm64` to keep cibuildwheel from complaining about having to skip testing the arm64 part of a universal2 wheel.

---

Looks like that change worked, there's now a universal2 wheel in the artifact. Need to find someone with a compatible Mac to test what pip does.

---

Well, by "worked" I mean it built `MarkupSafe-2.0.0a1-cp36-abi3-macosx_10_0_universal2.whl`, which hopefully only gets picked for 3.9 by pip...

---

10_0???

---

My bad, a typo 😅

---

Whew. :)